modules/claims_api/app/controllers/claims_api/v2/blueprints/claim_blueprint.rb

```diff
@@ -47,6 +47,23 @@ class ClaimBlueprint < Blueprinter::Base
         end
         field '5103_waiver_submitted'.to_sym
 
+        field :tracked_items do |claim, options|
+          bgs_service = BGS::Services.new(external_uid: options[:ptpct_id],
+                                          external_key: options[:ptpct_id])
+          bgs_response = bgs_service.tracked_items.find_tracked_items(claim[:claim_id])
+          bgs = bgs_response.dig(:benefit_claim, :dvlpmt_items) || []
```

Comment:
I kinda like the idea of this dig being in the `bgs-ext` gem, but other services didn't have it, so I presume that means it's up to `vets-api` to do the dig.